<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataDialog</class>
 <widget class="QDialog" name="MetadataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="PushButton" name="btn_accept">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="PushButton" name="btn_reject">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>633</width>
        <height>715</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Metadata&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="BodyLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>File Name</string>
             </property>
             <property name="buddy">
              <cstring>line_edit_file_name</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="LineEdit" name="line_edit_file_name">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="BodyLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>File Type</string>
             </property>
             <property name="buddy">
              <cstring>line_edit_file_type</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="LineEdit" name="line_edit_file_type">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="BodyLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>Sampling Rate*</string>
             </property>
             <property name="buddy">
              <cstring>spin_box_sampling_rate</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="SpinBox" name="spin_box_sampling_rate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string> Hz</string>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="BodyLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>Signal Column / Channel*</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="ComboBox" name="combo_box_signal_column">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="BodyLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>Info Column / Channel</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="ComboBox" name="combo_box_info_column">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="BodyLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>Additional Info</string>
             </property>
             <property name="buddy">
              <cstring>data_tree_widget_additional_info</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="DataTreeWidget" name="data_tree_widget_additional_info">
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="expandsOnDoubleClick">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>3</number>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">2</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">3</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>*Field is required</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>line_edit_file_name</tabstop>
  <tabstop>line_edit_file_type</tabstop>
  <tabstop>spin_box_sampling_rate</tabstop>
  <tabstop>combo_box_signal_column</tabstop>
  <tabstop>combo_box_info_column</tabstop>
  <tabstop>data_tree_widget_additional_info</tabstop>
  <tabstop>btn_accept</tabstop>
  <tabstop>btn_reject</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
